### PR TITLE
Merge Akka Typed to master branch

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by patriknw** [comment]
test failure is unrelated to Akka Typed as far as I can see, created https://github.com/akka/akka/issues/23001

**Comment by patriknw** [comment]
PLS BUILD

**Comment by ktoso** [comment]
Was just looking at it, a bit weird - yes certainly unrelated

File: akka-docs/rst/java/index-actors.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Actors
```
Comment (patriknw) [inline akka-docs/rst/java/index-actors.rst]: nothing in `akka-docs/rst/` should be added

File: akka-docs/src/main/paradox/scala/typed.md [hunk 2]
```diff
@@ -3,22 +3,22 @@
 @@@ warning
 
 This module is currently marked as @ref:[may change](common/may-change.md) in the sense
```
Comment (patriknw) [inline akka-docs/src/main/paradox/scala/typed.md]: there should be a java/typed.md also, I wrote that rst page and code examples (so it's only about converting it)

and it should be included in java/index-actors

**Review by patriknw — CHANGES_REQUESTED** [review]

File: akka-docs/rst/java/index-actors.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/java/index-actors.rst]: Whoops, merge mistake indeed - thanks!

File: akka-docs/src/main/paradox/scala/typed.md (hunk 2 above)
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/typed.md]: Ok cool, glad it already exists even :-)

File: project/MiMa.scala [hunk 3]
```diff
@@ -1158,41 +1158,63 @@ object MiMa extends AutoPlugin {
...
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getRouterConfigSerializerId"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getRouterConfigManifestBytes"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getConfigManifestBytes"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.hasConfigManifest"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.hasScopeSerializerId"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getRouterConfigManifest"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.hasConfigSerializerId"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getScopeSerializerId"),
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.WireFormats#DeployDataOrBuilder.getScopeManifest")
```
Comment (patriknw) [inline project/MiMa.scala]: not sure what happend here, perhaps not change anything in mima apart from the `watchWith` filter

**Review by patriknw — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM, the changes are in the expected files so I think it's good

**Pull request merged**